proto/system_config.proto

```diff
@@ -126,6 +126,9 @@ message DaqConfig {
 
   // Drop into console mode after test
   bool use_console = 52;
+
+  // Test vlan range
+  TestVLANRange test_vlan_range = 53;
```

Comment:
As mentioned previously, I think this should be renamed to something more generic.

(Also, as an aside this would be TestVlanRange since in this context it's equivalency between snake_case and CaMeL case that matters.)